.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
```

Comment:
@ahlag This looks like a copy of scikit-learn's bug report form. You're going to update it, right?

Reply:
@harupy 
It's still WIP. I am testing in locally on a different repository.

Reply:
Sounds good!